roles/puppet_repositories_bullseye/tasks/main.yml

```
@@ -0,0 +1,4 @@
+---
+- name: 'Install Puppet repository'
```

```suggestion
- name: 'Install Buster Puppet repository on Bullseye'
```

And include a comment with the ticket from puppetlabs?